deploy/olm-catalog/akka-cluster-operator/0.2.0/akka-cluster-operator.v0.2.0.clusterserviceversion.yaml

```diff
@@ -96,17 +96,6 @@ spec:
           - description: 'The template used to form the cluster'
             displayName: 'Template'
             path: template
-        statusDescriptors:
```

Comment:
Do you really want to remove these?  The actual values aren't duplicated in the other `statusDescriptors` bit.

Just get rid of the second `statusDescriptors` line?

Same comment in all other cases here...

Reply:
That's the catch. The values are duplicated, but under `specDescriptors`, hence it's a status-spec hybrid/duplicate.
To make the duplicate more direct, rename line 99 `specDescriptors`, then delete the duplicate `specDescriptors`.